validator/client/validator_metrics.go

```diff
@@ -64,10 +64,12 @@ func (v *validator) LogValidatorGainsAndLosses(ctx context.Context, slot uint64)
 			prevBalance := float64(v.prevBalance) / float64(params.BeaconConfig().GweiPerEth)
 			percentNet := (newBalance - prevBalance) / prevBalance
 			log.WithField("prevEthBalance", prevBalance).Infof("%v Previous validator balance", tpk)
-			log.WithFields(logrus.Fields{
-				"eth":           fmt.Sprintf("%f", newBalance-prevBalance),
-				"percentChange": fmt.Sprintf("%.2f%%", percentNet*100),
-			}).Infof("%v Net gains/losses in eth", tpk)
+			if v.logValidatorBalances {
+				log.WithFields(logrus.Fields{
+					"eth":           fmt.Sprintf("%f", newBalance-prevBalance),
+					"percentChange": fmt.Sprintf("%.2f%%", percentNet*100),
+				}).Infof("%v Net gains/losses in eth", tpk)
```

Comment:
Do not use `Infof`. I think this was changed in #2377

Reply:
I'll probably have to pull again. Since this what was pushed yesterday. The only thing I added was the conditional in that file

Reply:
What is the alternative to Infof?

Reply:
`Info`